solc/CommandLineInterface.cpp

```diff
@@ -1231,6 +1234,25 @@ void CommandLineInterface::outputCompilationResults()
 		handleNatspec(true, contract);
 		handleNatspec(false, contract);
 	} // end of contracts iteration
+
+	if (!g_HasOutputed &&
+			!m_args.count(g_argAst) &&
```

Comment:
Why do we still need those?

Reply:
Some information is not printed on screen when -o is given.